sig-cluster-lifecycle/annual-report-2021.md

```diff
@@ -106,17 +152,14 @@ In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $Y
 Manually visible via `git diff HEAD@{$YYYY-01-01} HEAD@{$YYYY-12-31} -- $sig-id/README.md`
 -->
 
-New in $YYYY:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+New in 2021:
+- NONE
 
-Retired in $YYYY:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+Retired in 2021:
+- [WG Component Standard](https://github.com/kubernetes/community/commit/a8fb89db5534f659e62e5c04528445b933d8e434)
 
 Continuing:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+- NONE
```

Comment:
forgot that we are sponsoring wg reliability:
https://github.com/kubernetes/community/blob/master/wg-reliability/README.md#stakeholder-sigs